lib/index.js

```diff
@@ -12,13 +12,8 @@ const versionSelectorMiddleware = specs => (req, res, next) => {
     return next();
   }
 
-  const mobileHeader = req.headers.mobile;
-  if (!mobileHeader) return next();
-
-  const result = lowerCase(mobileHeader, '').split('-');
-
-  const osName = result[0];
-  const buildNumber = Number.parseInt(result[1], 10);
+  const osName = lowerCase(req.headers['device-os']);
+  const buildNumber = Number.parseInt(req.headers['build-number'] || '0', 10);
```

Comment:
If the names of the headers are case sensitive please change to `Device-OS` and `Build-Number`

Reply:
@mtarekswvl nodejs converts all headers to lowercase